---
ros_pybullet_interface/ResetDebugVisualizerCamera